iOS-Email-Client/Views/es-419.lproj/RestoreUIPopover.strings

```diff
@@ -0,0 +1,12 @@
+
+/* Class = "UIButton"; normalTitle = "Restore from Cloud"; ObjectID = "CDQ-w1-eqI"; */
+"CDQ-w1-eqI.normalTitle" = "Restaurar desde iCloud";
+
+/* Class = "UILabel"; text = "Something about how to restore from Drive  you should give us permissions, and you can always restore later.\n\n"; ObjectID = "WNQ-UH-w8G"; */
```

Comment:
don't forget this translation

Reply:
It's a placeholder, string is overwritten in code